modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -1453,9 +1454,14 @@ private void _validateName(
 
 		ObjectField objectField = objectFieldPersistence.fetchByODI_N(
 			objectDefinition.getObjectDefinitionId(), name);
+		ObjectRelationship objectRelationship =
+			ObjectRelationshipLocalServiceUtil.
+				fetchObjectRelationshipByObjectDefinitionId(
+					objectDefinition.getObjectDefinitionId(), name);
```

Comment:
Could you add a empty line between lines 1456 and 1457, please?

Also, use `_objectRelationshipPersistence` instead `ObjectRelationshipLocalServiceUtil`.

Reply:
We agreed on using a `snapshot` for `ObjectRelationshipLocalService`